frame/proxy/src/lib.rs

```diff
@@ -304,22 +288,17 @@ pub mod pallet {
 		/// same sender, with the same parameters.
 		///
 		/// Fails if there are insufficient funds to pay for deposit.
-		///
-		/// # <weight>
-		/// Weight is a function of the number of proxies the user has (P).
-		/// # </weight>
-		/// TODO: Might be over counting 1 read
-		#[pallet::weight(T::WeightInfo::anonymous(T::MaxProxies::get()))]
-		pub fn anonymous(
+		#[pallet::weight(T::WeightInfo::pure(T::MaxProxies::get()))]
+		pub fn pure(
```

Comment:
Every other dispatchable function is a verb, for me at least it's a sensible pattern to follow.
```suggestion
		pub fn create_pure(
```